### PR TITLE
Chore: Enable PDF signatures in future builds of PDF.js

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: build/upgrade_pdfjs.sh [hunk 1]
```diff
@@ -35,3 +35,12 @@ echo "--------------------------------------------------------------------------
...
 sed -e 's@var DEFAULT_CACHE_SIZE = 10;@var DEFAULT_CACHE_SIZE = /iphone|ipad|ipod|android|blackberry|bb10|mini|windows\sce|palm/i.test(navigator.userAgent) ? 5 : 10;@' -i '' src/third-party/doc/$DOC_STATIC_ASSETS_VERSION/pdf_viewer.js
+
+
+# Deletes code hiding PDF signatures.
+# For preview purposes, it is acceptable to show signatures without verifying them.
+echo "-----------------------------------------------------------------------------------"
+echo "Enabling PDF signatures"
+echo "-----------------------------------------------------------------------------------"
+# Deletes a block of code that starts with `if (data.fieldType === 'Sig') {``, and ends with `}`
+sed -e "/if (data.fieldType === 'Sig') {/,/}/d" -i '' src/third-party/doc/1.30.0/pdf.worker.js
```
Comment (DanDeMicco) [inline build/upgrade_pdfjs.sh]: Should you use ```DOC_STATIC_ASSETS_VERSION```?

Comment (jeremypress) [reply build/upgrade_pdfjs.sh]: yep, forgot to put it back after testing.

**Comment by DanDeMicco** [comment]
Should we add some safeguards to this file to make it so if any of these modifications fails to do something, it fails the build?

**Pull request merged**